charts/modules/apps/crossplane/README.md

```diff
@@ -58,14 +60,14 @@
-| ------------------------------------------------------------------------------ | ---------------------------------------------------------- | ----------------------------------------------- |
-| `crossplane-aws-iam.enabled`                                                   | Toggle for enabling or disabling upstream chart templates. | `false`                                         |
-| `crossplane-aws-iam.Role.items._.annotations.crossplane.io/external-name`      | AWS resource name to import if exists.                     | `crossplane-provider-aws`                       |
-| `crossplane-aws-iam.Role.items._.deletionPolicy`                               | Deletion policy for the role.                              | `Orphan`                                        |
-| `crossplane-aws-iam.RolePolicyAttachment.items._.deletionPolicy`               | Deletion policy for the role.                              | `Orphan`                                        |
-| `crossplane-aws-iam.RolePolicyAttachment.items._.forProvider.policyArn`        | Default administratorAccess Policy resource ARN name.      | `arn:aws:iam::aws:policy/AdministratorAccess`   |
-| `crossplane-aws-iam.RolePolicyAttachment.items._.forProvider.roleNameRef.name` | Reference to the Role resource created by this helm chart. | `{{ include "common-gitops.names.release" . }}` |
+| Name                                                                                              | Description                                                | Value                                                        |
+| ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- | ------------------------------------------------------------ |
+| `crossplane-aws-iam.enabled`                                                                      | Toggle for enabling or disabling upstream chart templates. | `false`                                                      |
+| `crossplane-aws-iam.Role.items.provider-aws.annotations.crossplane.io/external-name`              | AWS resource name to import if exists.                     | `crossplane-provider-aws`                                    |
+| `crossplane-aws-iam.Role.items.provider-aws.deletionPolicy`                                       | Deletion policy for the role.                              | `Orphan`                                                     |
+| `crossplane-aws-iam.RolePolicyAttachment.items.provider-aws-adminaccess.deletionPolicy`           | Deletion policy for the role.                              | `Orphan`                                                     |
+| `crossplane-aws-iam.RolePolicyAttachment.items.provider-aws-adminaccess.forProvider.policyArn`    | Default administratorAccess Policy resource ARN name.      | `arn:aws:iam::aws:policy/AdministratorAccess`                |
```

Comment:
Need to think about this as a default permission. Can you please add a comment to review this later?

Initial thoughts: I think we could create a custom policy that is automatically adds statements for each provider family that we add. This will make SecOps folks happy.